app/src/main/java/com/github/palFinderTeam/palfinder/ui/login/LoginActivity.kt

```diff
@@ -42,6 +43,7 @@ import com.google.firebase.ktx.Firebase
 import kotlinx.coroutines.launch
 
 const val CREATE_ACCOUNT_PROFILE = "com.github.palFinderTeam.palFinder.CREATE_ACCOUNT_PROFILE"
+var IS_NO_ACCOUNT_USER = false
```

Comment:
Not necessary a good practice, maybe you could put it as a member of ProfileService and have way to set the service in no account mode or not.
On the other hand this is a simple solution.

Reply:
you could even try if simply checking that currentLoggedInUser == null <=> IS_NO_ACCOUNT_USER